content/_index.md

```diff
@@ -7,16 +7,16 @@ type: docs
 A home for compiler team planning documents, meeting minutes, and
 other such things. If you're interested in learning about how
 **rustc** works -- as well as advice on building the compiler, preparing a PR,
-and other similar topics -- check out the [rustc-guide].
+and other similar topics -- check out the [rustc-dev-guide].
 
-[rustc-guide]: https://rust-lang.github.io/rustc-guide/
+[rustc-dev-guide]: https://rust-lang.github.io/rustc-dev-guide/
 
 ## Quick facts
 
 - **Where to find us:** We are primarily present on [Zulip](about/chat-platform), though some of us also monitor Discord.
 - **Want to attend a meeting or follow along?** See the "meeting calendar" section below.
 - **Looking for technical information about how the compiler works?**
-  Check out the [rustc-guide](https://rust-lang-nursery.github.io/rustc-guide/).
+  Check out the [rustc-dev-guide](https://rust-lang-nursery.github.io/rustc-dev-guide/).
```

Comment:
Can we update this to avoid redirection by the nursery?

Reply:
```suggestion
  Check out the [rustc-dev-guide](https://rust-lang.github.io/rustc-dev-guide/).
```

Reply:
fixed in https://github.com/rust-lang/compiler-team/pull/254/commits/a4f982f886752e6a9f3c7d9b6c74eaed1c6c780c. ty!